cylc/flow/scripts/tui.py

```diff
@@ -89,6 +93,12 @@ def main(_, options: 'Values', workflow_id: str) -> None:
         workflow_id,
         constraint='workflows',
     )
+
+    # Throw out workflow_id's referring to not-installed workflows.
+    if not is_installed(get_workflow_run_dir(workflow_id)):
+        LOG.error(f'{workflow_id} is not an installed workflow')
+        exit(1)
```

Comment:
This disables Tui for manually installed workflows, this needs to be solved in `cylc.flow.tui.app` (look for `WorkflowStopped`)